modules/ivc_champva/app/form_mappings/vha_10_10d.json.erb

```diff
@@ -11,60 +11,60 @@
   "form1[0].#subform[0].VeteransPhoneNumber[0]":         "<%= form.data.dig('veteran', 'phone_number') %>",
   "form1[0].#subform[0].VeteransDateOfBirth[0]":         "<%= form.data.dig('veteran', 'date_of_birth') %>",
   "form1[0].#subform[0].VeteransDateOfMarriage[0]":      "<%= form.data.dig('veteran', 'date_of_marriage') %>",
-  "form1[0].#subform[0].IsTheVeteranDeceased[0]":        "<%= form.data.dig('veteran', 'is_deceased') ? 1 : 0  %>",
+  "form1[0].#subform[0].IsTheVeteranDeceased[0]":        "<%= form.data.dig('veteran', 'sponsor_is_deceased') ? 0 : 1  %>",
   "form1[0].#subform[0].VeteransDateOfDeath[0]":         "<%= form.data.dig('veteran', 'date_of_death') %>",
-  "form1[0].#subform[0].DieOnActiveMilitaryService[0]":  "<%= form.data.dig('veteran', 'is_active_service_death') ? 1 : 0  %>",
+  "form1[0].#subform[0].DieOnActiveMilitaryService[0]":  "<%= form.data.dig('veteran', 'is_active_service_death') ? 0 : 1  %>",
   "form1[0].#subform[0].LastName1[0]":                   "<%= form.data['applicants'][0]&.dig('full_name', 'last') %>",
   "form1[0].#subform[0].FirstName1[0]":                  "<%= form.data['applicants'][0]&.dig('full_name', 'first') %>",
-  "form1[0].#subform[0].MI1[0]":                         "<%= form.data['applicants'][0]&.dig('full_name', 'middle') %>",
+  "form1[0].#subform[0].MI1[0]":                         "<%= form.data['applicants'][0]&.dig('full_name', 'middle')&.first %>",
   "form1[0].#subform[0].SSN1[0]":                        "<%= form.data['applicants'][0]&.dig('ssn_or_tin') %>",
   "form1[0].#subform[0].StreetAddress1[0]":              "<%= form.data['applicants'][0]&.dig('address', 'street') %>",
   "form1[0].#subform[0].City1[0]":                       "<%= form.data['applicants'][0]&.dig('address', 'city') %>",
   "form1[0].#subform[0].ZipCode1[0]":                    "<%= form.data['applicants'][0]&.dig('address', 'postal_code') %>",
   "form1[0].#subform[0].DateOfBirth1[0]":                "<%= form.data['applicants'][0]&.dig('date_of_birth') %>",
   "form1[0].#subform[0].EmailAddress1[0]":               "<%= form.data['applicants'][0]&.dig('email') %>",
   "form1[0].#subform[0].PhoneNumber1[0]":                "<%= form.data['applicants'][0]&.dig('phone_number') %>",
-  "form1[0].#subform[0].Gender1[0]":                     "<%= form.data['applicants'][0]&.dig('gender') ? 1 : 0 %>",
+  "form1[0].#subform[0].Gender1[0]":                     "<%= form.data['applicants'][0]&.dig('gender') == 'male' ? 0 : 1 %>",
```

Comment:
Fixes bug where if gender was set at all, it was set to male and female otherwise.